config/routes.rb

```diff
@@ -1,5 +1,8 @@
 Rails.application.routes.draw do
+  get 'users/new'
```

Comment:
is this specifying our homepage? i.e. is this saying that the first view our users will see is the login/signup page?

Reply:
At the moment, it looks like it's our homepage. It has been set up by default. We will need to create an 'index controller' and probably move some codes to this controller or redirect there